src/tools/dataValidation.test.js

```diff
@@ -62,8 +62,47 @@ describe('dataValidation', () => {
     });
   });
 
-  it('should log results to the auditLogger', async () => {
+  it('should log specific messages to the auditLogger', async () => {
     await dataValidation();
-    expect(auditLogger.info).toHaveBeenCalledTimes(10);
+
+    const complexPatterns = [
+      /Grants data counts: \[\s*(.|\s)*\s*\]/m,
+      /ActivityReports data counts: \[\s*(.|\s)*\s*\]/m,
+    ];
+
+    const simplePatterns = [
+      /Attempting to connect to the database: {}/,
+      /Database connection established: {}/,
+      /Attempting to disconnect from the database: {}/,
+      /Database connection closed: {}/,
```

Comment:
If we ever change the verbiage in `src/models/index`, this seemingly unrelated test will fail, which seems not great. Can we refactor this to remove the connection-related logging, and only test the `...last updated at...` stuff?